src/error.rs

```diff
@@ -24,17 +22,6 @@ impl fmt::Display for Error {
     }
 }
 
-impl StdError for Error {
-    fn description(&self) -> &str {
-        match *self {
-            Error::Format => "invalid formatting",
-            Error::Base64(ref x) => x.description(),
-            Error::Json(ref x) => x.description(),
-            Error::Utf8(ref x) => x.description(),
-        }
-    }
-}
-
```

Comment:
With this change you completely removed the implementation of the `std::error::Error` trait.
To only fix the problem with the `description` function being deprecated you could replace it with:
```
impl std::error::Error for Error {}
```

Reply:
I created #18 to fix this.